core/templates/dev/head/domain/question/QuestionObjectFactory.ts

```diff
@@ -149,8 +151,10 @@ angular.module('oppia').factory('QuestionObjectFactory', [
       var questionBackendDict = {
         id: null,
         question_state_data: this._stateData.toBackendDict(),
+        question_state_data_schema_version: CURRENT_STATE_SCHEMA_VERSION,
```

Comment:
Yeah, the schema version is not surfaced in the frontend at any time. In `QuestionCreationHandler`, we add the schema version to the dict, only in the backend, so this shouldn't be here.
@sagangwee Was there any particular reason to add this?

Reply:
Yes, when the question suggestion is added, it will fail validation if the passed question_dict does not have the necessary data [here](https://github.com/oppia/oppia/blob/develop/core/domain/suggestion_registry.py#L542).

Reply:
Oh, then is it possible to change the backend controller to not depend on the frontend for schema version, similar to `QuestionCreationHandler`? Since, we generally don't surface schema versions in the frontend.